packages/common/src/utils/formatUtil.test.ts

```diff
@@ -0,0 +1,151 @@
+import { AUDIO, FixedDecimal, USDC, wAUDIO } from '@audius/fixed-decimal'
+import BN from 'bn.js'
+
+import { BNWei } from 'models/Wallet'
+
+import {
+  formatCurrencyBalance,
+  formatAudio,
+  formatWeiToAudioString,
+  formatPrice,
+  convertFloatToWei,
+  formatNumberString,
+  formatCount,
+  formatBytes,
+  parseWeiNumber
+} from './formatUtil'
+
+describe('formatUtil', function () {
+  it('can format count', function () {
```

Comment:
added count here but other than that stuck to only functions relevant to `FixedDecimal`